examples/atlas-streams/stream-connection/sample-stream-connection.json

```diff
@@ -0,0 +1,38 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "This template creates one connection of type 'Sample' with the sample_stream_solar dataset used",
+  "Parameters": {
+    "Profile": {
+      "Type": "String",
+      "Default": "default",
+      "Description": "Secret Manager Profile that contains the Atlas Programmatic keys"
+    },
+    "ProjectId": {
+      "Type": "String",
+      "Description": "Atlas Project Id"
+    },
+    "InstanceName": {
+      "Type": "String",
+      "Description": "Human-readable label that identifies the stream instance"
+    }
+  },
+  "Mappings": {},
+  "Resources": {
+    "StreamConnection": {
+      "Type": "MongoDB::Atlas::StreamConnection",
+      "Properties": {
+        "ProjectId": {
+          "Ref": "ProjectId"
+        },
+        "Profile": {
+          "Ref": "Profile"
+        },
+        "InstanceName": {
+          "Ref": "InstanceName"
+        },
+        "Type": "Sample",
+        "ConnectionName": "sample_stream_solar"
```

Comment:
in the example don't we want to allow to choose connection name (sample source) ? it can have a default value, but allow them to change

Reply:
Uncertain about this.
Currently, any other value than `sample_stream_solar` will fail.
But I think you are right. It will be better for the future. 

The alternative of one example per stream_connection sample dataset, but that might be overkill.

Reply:
Changed in 2ce1217